addons/campsite/helper/fn_canBuildCampsite.sqf

```diff
@@ -0,0 +1,32 @@
+#include "script_component.hpp"
+/*
+ * Author: Gilles
+ *
+ * TODO DOCSTRING
+ *
+ * Arguments:
+ * 0: xyz
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ *
+ * Public: No
+ *
+ * Scope: Client
+ */
+
+params["_structure"];
```

Comment:
I do not fully understand the term "_structure" in combination with the 0 and 1 switch case options. Try to make it more clear at first glance

Reply:
insted of making a lot of functions for each "camping structure", I'm using and ID for each kind. 0 = camping area, 1 = tent, 2 = store area and so on.